README.md

```
@@ -154,6 +154,7 @@ Name | Type | Description
 [redhatci.ocp.junit2dict]() | Filter | Transforms a JUnit into a dictionary
 [redhatci.ocp.junit2obj]() | Filter | Transforms a JUnit XML into a corresponding JSON text
 [redhatci.ocp.ocp_compatibility]() | Filter | Parse the deprecated and to-be-deprecated API after the workload installation
+[redhatci.ocp.reportsmerger]() | Filter | Merges multiple similarly formatted JSON test reports into 1
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)